src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Runtime.InteropServices;
+using System.IO;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibGpiodDriver : UnixDriver
```

Comment:
Given that both this and the SysFSDriver are public, you will need to also have to provide  a Windows implementation for them which just throws PNS Exception

Reply:
Do we want to seal these new classes?

Reply:
By my understanding the API is not intending to expose them, instead decide which driver to load runtime. By your suggestion I made them public as libgpiod driver event handling is not done yet, but as when event handling is implemented we gonna make them internal back I think we don't need to provide Windows implementation

Reply:
@ahsonkhan we currently don't bother with sealing stuff, there is too high chance we will unseal it in the future (i.e. new version of libgpiod shows up with the new kernel)

Reply:
Sounds good to me. However, I am wondering why would a new version of libgpiod require us to unseal the class?

Reply:
well i don't think new version of libgpiod would require to unseal, never heard something like require to be unsealed, instead heard required to be sealed. So i would like to ask why we need to seal the class. I saw classes sealed only if they really need to be sealed or required to be sealed